cmd/kubeapps-apis/proto/kubeappsapis/plugins/helm/packages/v1alpha1/helm.proto

```diff
@@ -207,3 +219,54 @@ message RepositoryFilterRule {
   // map of variables
   map<string, string> variables = 4;
 }
+
+// ProxyOptions
+//
+// TODO
+message ProxyOptions {
+  // TODO
+  bool enabled = 1;
+
+  // TODO
+  string http_proxy = 2;
+
+  // TODO
+  string https_proxy = 3;
+
+  // TODO
+  string no_proxy = 4;
+}
+
+
+
+// Toleration
+//
+// Extracted from the K8s API to avoid a dependency on the K8s API
+// https://github.com/kubernetes/api/blob/master/core/v1/generated.proto
+message Toleration {
+	optional string key = 1;
+	optional string operator = 2;
+	optional string value = 3;
+	optional string effect = 4;
+	optional int64 toleration_seconds = 5;
+}
+
+
+// PodSecurityContext
+//
+// Extracted from the K8s API to avoid a dependency on the K8s API
+// https://github.com/kubernetes/api/blob/master/core/v1/generated.proto
+message PodSecurityContext {
+  optional int64 run_as_user = 1;
+  optional int64 run_as_group = 6;
+  optional bool run_as_non_root = 3;
+  repeated int64 supplemental_groups = 4;
+  optional int64 f_s_group = 5;
+
+  // TODO(agamez): more complex fields are not supported yet
+  // optional SELinuxOptions seLinuxOptions = 1;
+  // optional WindowsSecurityContextOptions windowsOptions = 8;
+  // repeated Sysctl sysctls = 7;
+  // optional string fsGroupChangePolicy = 9;
+  // optional SeccompProfile seccompProfile = 10;
+}
```

Comment:
As I said in the PR description, I only kept the most simple fields. This is preventing us from importing the whole proto document from k8s.